.github/workflows/release.yaml

```diff
@@ -33,45 +33,36 @@ jobs:
           go-version: 'stable'
 
       # This step is only used for testing in the official kangaroo repo
-      - name: Overwrite Names for release testing and fetch kando webhapp from github
-        if: ${{ github.repository }} == 'holochain-apps/holochain-kangaroo-electron'
+      - name: Overwrite Names for release testing
+        if: ${{ github.repository }} == 'holochain-apps/kangaroo-electron'
         run: |
           echo ${{ github.repository }}
           echo "overwriting names for release testing"
-          curl -f -L --output ./pouch/ziptest.webhapp https://github.com/holochain/ziptest/releases/download/ziptest-v0.1.0/ziptest.webhapp
           node ./scripts/overwrite-with-test-name.js
 
       - name: Environment setup
         run: |
           yarn setup
 
-      - name: Retrieve version
-        run: |
-          echo "Retrieved App version: $(node -p -e "require('./package.json').version")"
-          echo "APP_VERSION=$(node -p -e "require('./package.json').version")" >> $GITHUB_OUTPUT
-        id: version
+      - name: Read kangaroo config
         shell: bash
-
-      - name: Retrieve appId
+        id: kangarooConfig
         run: |
+          echo "APP_VERSION=$(node -p -e "require('./package.json').version")" >> $GITHUB_OUTPUT
           echo "APP_ID=$(node ./scripts/read-app-id.js)" >> $GITHUB_OUTPUT
-        id: appId
-        shell: bash
-
-      - name: Retrieve whether Windows code signing should be attempted
-        run: |
+          echo "MACOS_CODE_SIGNING=$(node ./scripts/read-macos-code-signing.js)" >> $GITHUB_OUTPUT
           echo "WINDOWS_CODE_SIGNING=$(node ./scripts/read-windows-code-signing.js)" >> $GITHUB_OUTPUT
-        id: shouldWindowsCodeSign
-        shell: bash
 
-      - name: Retrieve whether macOS code signing should be attempted
-        run: |
-          echo "MACOS_CODE_SIGNING=$(node ./scripts/read-macos-code-signing.js)" >> $GITHUB_OUTPUT
-        id: shouldMacOSCodeSign
+      - name: Check that webhapp is available in pouch folder
```

Comment:
This is a new check to see if there is a webhapp in the pouch folder so that devs get a meaningful error if not.